pkg/config/config.go

```diff
@@ -44,6 +44,7 @@ var defaultReplicaConfig = &ReplicaConfig{
 		Tp:          "table-number",
 		PollingTime: -1,
 	},
+	KVClientV2: true,
```

Comment:
```suggestion
	KVClientV2: false,
```
Remember to turn off `kv-client-v2` before PR merged